resources/lib/downloadutils.py

```diff
@@ -13,19 +13,19 @@
 from kodi_utils import HomeWindow
 from clientinfo import ClientInformation
 from simple_logging import SimpleLogging
+from translation import i18n
 
-log = SimpleLogging("EmbyCon." + __name__)
+log = SimpleLogging(__name__)
 
-class DownloadUtils():
 
+class DownloadUtils():
     getString = None
 
     def __init__(self, *args):
-        settings = xbmcaddon.Addon(id='plugin.video.embycon')
-        self.getString = settings.getLocalizedString
-        self.addon_name = settings.getAddonInfo('name')
+        self.settings = xbmcaddon.Addon(id='plugin.video.embycon')
```

Comment:
dont store settings, always create them when you need them, creating takes a snapshot of the settings and if the settings are stored and life cycled with the class you dont know how old the settings were, they could be stale.